playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/Program.cs

```diff
@@ -3,7 +3,10 @@
 
 var builder = DistributedApplication.CreateBuilder(args);
 
-var openai = builder.AddConnectionString("openai");
+builder.AddAzureProvisioning();
+
+var openai = builder.AddAzureOpenAI("openai");
+openai.AddDeployment("gpt-35-turbo", "gpt-35-turbo", "0613");
```

Comment:
No fluent API?

Reply:
Do you know of a resource I should use as an example?